src/components/organisms/stories/group-policy-form.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryFn } from '@storybook/react'
+
+import { defaultGroupPolicyFormValues } from 'util/form.defaults'
+
+import { GroupPolicyForm } from '../group-policy-form'
+
+// TODO: need to mock Wallet store for this to render
```

Comment:
Are we tracking this anywhere? Maybe could be done as part of #7.

Reply:
Oh good catch - this comment is actually outdated. Moving storybook off the alpha version (which I did in https://github.com/regen-network/groups-ui/pull/40) fixed this issue - removing comment now 👍